rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2234,8 +2262,8 @@ bool opcode_00c2(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloa
     opcode 00c3
 */
 void opcode_00c3(const ScriptArguments& args) {
-    RW_UNIMPLEMENTED_OPCODE(0x00c3);
-    RW_UNUSED(args);
+    args.getVM()->setDebugFlag(true);
+
```

Comment:
What does this even do? Are we sure it should be part of the SCM interpreter / `ScriptMachine`?
Maybe this debug flag is for the gamestate instead? Is there any documentation for this?

Reply:
This is part of SCM interpreter. Though, this is disabled in original game and completely removed from Vice City. It is still functional if you patch 1 asm instruction. IIRC, it is used to draw boundaries for locate opcodes.

Reply:
Still weird that it's part of the `ScriptMachine` in my opinion.
Now that I know what it does, I think we should definitely implement it.

*(Although we should probably also expose it via the debug menu later, which could be a follow-up issue)*

Reply:
'''
float dist  = std::abs(coord.x - node->position.x);
      dist += std::abs(coord.y - node->position.y);
      dist += std::abs(coord.z - node->position.z) * 3.f;